runtime/drivers/sqlite/registry.go

```diff
@@ -63,15 +63,16 @@ func (c *connection) CreateInstance(_ context.Context, inst *drivers.Instance) e
 	now := time.Now()
 	_, err := c.db.ExecContext(
 		ctx,
-		"INSERT INTO instances(id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on) "+
-			"VALUES ($1, $2, $3, $4, $5, $6, $7, $7)",
+		"INSERT INTO instances(id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on, env) "+
+			"VALUES ($1, $2, $3, $4, $5, $6, $7, $7, $8)",
```

Comment:
With the env as a `map[string]string`, for simplicity, I would suggest serializing/deserializing it as JSON here (keeping the column type as `TEXT` – sqlite's type support can sometimes act unexpected).

Reply:
(Requires changes in `findInstances` function as well)